packages/runtime-dom/src/directives/vModel.ts

```diff
@@ -239,11 +239,6 @@ function setSelected(
     return
   }
 
-  // fast path for updates triggered by other changes
-  if (isArrayValue && looseEqual(value, oldValue)) {
```

Comment:
We could leave this in and guard it with `oldValue !== value`

That would allow to opt into the fastpath by using an immutable array.

@yyx990803 WDYT?

Reply:
If users use `reactive` to contain an array, and change it's reference later then this will cause unexpected skipping update. [e.g.](http://127.0.0.1:5173/#eNqNVUtu2zAQvcpAKGAZkGWk2Smy0TZIgXTRFE27irJQpFHChKIEklJdGLpBr9DL9SQdkvrVdYLAC3O+782H1N57X9dh26AXebHKJKs1KNRNvU0EACvrSmrYSywCyKqybjTmAUhMM81aDOAe9XkjJQp9KZRORYbQQSGrEhaUc2FyZBVZYG+OJOhdNJwB3hSVzPB7nacaA6fszF8HmyOZ/eWUrqo1owP5ETX/JhGJdr+9SEuMYPGRSaUXAbQpb4x8sugI4dDrGildPnN7O3O7neEp5JhR7RbQFe9TGcatD725tcz/i7ngWFIRsPFJNKclbLamBQ5kSBzaNJT+JgzDf5UB9JFOtDgAiZ53z/Qm0bZ5DjzHA3h4DX6oas6o0wdaJnLcXRVDBqddBnCyPHN5ZsBD6NU4oWFv/KIR1LlK+MsJX9KuSTHMs8crGNcoBzjD91WEYAuriRJ1JF67jaZdJkFjWXPqld3sOGctsHyTeGldJ57x0OdU+xM8oMTIiHHDIeOpUuTk8A3F3pfMnEG7ohmQ2dEHNhaSeBA94c/R5BiS9l1mQMaM/Xh857Wk5Pt9nyM0OwpdB74NWcZrzix0vG64PVwPS9mDOtaqTsURYgdzeY7gEfw4nVgfrNXEGx7oIpLDY9qmrunRGWmH/kksq9ak//P7V7xOtzQZounKuZOw3tJCW/I2OxhoimoUSkUXy4Oy4ZrV3CiHI2nbVVnlyMduDvsxzaiv/6U5tas7WqfIBr6iHfHayf0oHDAJ8Zo2yv7P9swLPK3oXhTsPnxUlaBH1j5+iWcuBeMox3GMzyJtJOfVj09WpyXd/0GfPWD2dET/qHZGl3hfJI1Hmj6PNp1Kekmd+eL6M+7oPBqpe41p5AvGr6gq3hiOzu1DI3KiPfOzbC/th4KJ+2/qYqdRqKEoQ9Q+jNY/8eircP5C6RPd0/DUxiWi87q/wnxSzw==)